src/main/kotlin/com/cognifide/gradle/aem/common/pkg/PackageFileFilter.kt

```diff
@@ -62,13 +62,11 @@ class PackageFileFilter(private val task: PackageCompose) : Serializable {
         aem.prop.string("package.fileFilter.bundlePath")?.let { set(it) }
     }
 
-    fun filter(spec: CopySpec, expandProperties: Map<String, Any> = mapOf()) {
+    fun filter(spec: CopySpec) {
         if (excluding.get()) {
             spec.exclude(excludeFiles.get())
         }
 
-        val expandPropertiesAll = expandProperties + this.expandProperties.get() + mapOf("aem" to aem)
```

Comment:
this was causing eager evaluation of properties (in config phase, not in execution phase)

Reply:
should improve performance a little bit as of properties are lazy calculated then reused